.env.example

```
@@ -87,7 +87,7 @@ NEARAI_AUTH_URL=https://private.near.ai
 # CLI is always enabled
 
 # Slack Bot (optional)
-SLACK_BOT_TOKEN=xoxb-...
+SLACK_BOT_TOKEN=slack-bot-token-EXAMPLE
 SLACK_APP_TOKEN=xapp-...
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SLACK_APP_TOKEN key should go before the SLACK_BOT_TOKEN key

(UnorderedKey)